src/mapping/mapping_main.py

```diff
@@ -0,0 +1,125 @@
+"""The main file for the mapping module."""
+import logging
+import pandas as pd
+from typing import Callable
+
+from src.mapping import mapping_helpers as hlp
+from src.staging import staging_helpers as stage_hlp
+from src.staging import validation as val
+
+MappingMainLogger = logging.getLogger(__name__)
+
+
+def run_mapping(
+    full_responses,
+    config: dict,
+    check_file_exists: Callable,
+    load_json: Callable,
+    read_csv: Callable,
+    write_csv: Callable,
+    read_feather: Callable,
+    write_feather: Callable,
+    isfile: Callable,
```

Comment:
shall we import these directly into this module, rather than passing them?

Reply:
Ah, I think these being passed, rather than imported, because they will change depending on whether it is local v hdfs?

Reply:
yes, I think we were talking about simplifying it to     
# Check the environment switch
    network_or_hdfs = config["global"]["network_or_hdfs"]

    if network_or_hdfs == "network":
        from src.utils import local_file_mods as mods

    elif network_or_hdfs == "hdfs":
        from src.utils import hdfs_mods as mods

what do you think?

Reply:
Yes, you're right - we spoke about doing that in the module! Sorry, had slipped my mind